### PR TITLE
fix: override cli

### LINKED ISSUE #297: feat(config): Add support for nested chain config override
Right now I am only able to use `--chain-config-override` for the parent fields of the chain config like:

```
 --chain-config-override sequencer_address=0x123
```

I would like to be able to modify nested fields in order for exemple to modify the steps number of the chain config like:

```
--chain-config-override bouncer_config.block_max_capacity.n_steps=100000
```

It apears that [this function](https://github.com/madara-alliance/madara/blob/main/crates/node/src/cli/chain_config_overrides.rs#L21) does not support it

### PR DESCRIPTION
**Pull request by jbcaron**
# Pull Request type

- Bugfix

## What is the current behavior?

Resolves: #297 

## What is the new behavior?

- add default versioned_constants
- fix: override_chain_config

## Does this introduce a breaking change?

No

### CODE REVIEW

File: configs/presets/integration.yaml [hunk 1]
```diff
@@ -30,5 +26,5 @@ bouncer_config:
     message_segment_length: 18446744073709551615
     n_events: 18446744073709551615
     state_diff_size: 131072
-sequencer_address: "0x0"
+sequencer_address: "0x1176a1bd84444c89232ec27754698e5d2e7e1a7f1539f12027f28b23ec9f3d8"
```
Comment (antiyro) [inline configs/presets/integration.yaml:28]: where those this address comes from?

File: configs/presets/mainnet.yaml [hunk 2]
```diff
@@ -30,5 +26,5 @@ bouncer_config:
     message_segment_length: 18446744073709551615
     n_events: 18446744073709551615
     state_diff_size: 131072
-sequencer_address: "0x0"
+sequencer_address: "0x1176a1bd84444c89232ec27754698e5d2e7e1a7f1539f12027f28b23ec9f3d8"
```
Comment (antiyro) [inline configs/presets/mainnet.yaml:28]: Same here

File: configs/presets/sepolia.yaml [hunk 3]
```diff
@@ -30,5 +26,5 @@ bouncer_config:
     message_segment_length: 18446744073709551615
     n_events: 18446744073709551615
     state_diff_size: 131072
-sequencer_address: "0x0"
+sequencer_address: "0x1176a1bd84444c89232ec27754698e5d2e7e1a7f1539f12027f28b23ec9f3d8"
```
Comment (antiyro) [inline configs/presets/sepolia.yaml:28]: and here

File: crates/node/src/cli/chain_config_overrides.rs [hunk 4]
```diff
@@ -1,73 +1,51 @@
...
+    deserialize_bouncer_config, deserialize_starknet_version, serialize_bouncer_config, serialize_starknet_version,
+    ChainConfig, StarknetVersion,
+};
+use mp_utils::parsers::parse_key_value_yaml;
+use mp_utils::serde::{deserialize_duration, serialize_duration};
 
 /// Override chain config parameters.
-/// Format: "--chain-config-override chain_id=NEW_MADARA --chain-config-override chain_name=NEW_NAME"
-#[derive(clap::Parser, Clone, Debug)]
+/// Format: "--chain-config-override chain_id=SN_MADARA,chain_name=MADARA,block_time=1.5"
```
Comment (akhercha) [inline crates/node/src/cli/chain_config_overrides.rs]: `block_time=1.5` is only a typo right? It should be "1s", "500ms" etc...

Comment (jbcaron) [reply crates/node/src/cli/chain_config_overrides.rs]: yes that was before using our own `deserialize_duration` method

**Review by antiyro — APPROVED** [review]

**Review by akhercha — APPROVED** [review]
lgtm, thanks!

**Pull request merged**